Block/NodeType/Product.php

```diff
@@ -105,13 +116,13 @@ public function fetchData(array $nodes)
     {
         $storeId = $this->_storeManager->getStore()->getId();
 
-        list(
+        [
```

Comment:
This is a feature which can be used since PHP 7.1: https://www.php.net/manual/en/migration71.new-features.php#migration71.new-features.symmetric-array-destructuring

However the composer.json of the module points as minimum requirement Magento Framework version `100.1.*` (https://github.com/SnowdogApps/magento2-menu/blob/develop/composer.json#L21), which can be compatible with PHP 5.6 and PHP 7.0, so I think we should keep the `list` way to avoid issues with compatibilities.